api/src/main/java/gov/cms/ab2d/api/controller/v2/BulkDataAccessAPIV2.java

```diff
@@ -37,6 +37,7 @@
 import static gov.cms.ab2d.api.controller.common.ApiText.BULK_RESPONSE;
 import static gov.cms.ab2d.api.controller.common.ApiText.BULK_RESPONSE_LONG;
 import static gov.cms.ab2d.api.controller.common.ApiText.BULK_SINCE;
+import static gov.cms.ab2d.api.controller.common.ApiText.BULK_SINCE_DEFAULT;
```

Comment:
What is wrong with just doing a .* import?  After a certain number of imports, that makes sense to me.  There are several places where that is.  We should actually agree on a standard for the team as I constantly see the effects in code reviews of different IntelliJ settings, making for messier git history.